src/tilecache.js

```diff
@@ -63,13 +63,19 @@ ImageRecord.prototype = {
     },
 
     getRenderedContext: function() {
+        if (!this._renderedContext) {
+            var canvas = document.createElement( 'canvas' );
+            canvas.width = this._image.width;
+            canvas.height = this._image.height;
+            this._renderedContext = canvas.getContext('2d');
+            this._renderedContext.drawImage( this._image, 0, 0 );
+            //since we are caching the prerendered image on a canvas
+            //allow the image to not be held in memory
+            this._image = null;
+        }
         return this._renderedContext;
     },
 
-    setRenderedContext: function(renderedContext) {
```

Comment:
Yeah, this is pretty internal, so losing it probably isn't a big deal. Still, probably best to deprecate it (i.e. leave the function but have it $.console.error, and of course write it up in the changelog). You could even leave the code in it, really.